docs/pages/access-controls/getting-started.mdx

```diff
@@ -3,13 +3,12 @@ title: Getting Started With Access Controls
 description: Get started using Teleport Access Controls.
 ---
 
-# Getting Started
-
 In Teleport, any local, SSO, or robot user can be a member of one or several
```

Comment:
Not sure I love this working.

A user isn't really a "member" of a role.
A user _has_ one or more roles assigned to them.